content/en/docs/collector/trace-receiver.md

```diff
@@ -275,15 +277,17 @@ func (cfg *Config) Validate() error {
 }
 ```
 
-> #### Check your work
->
-> - I imported the `fmt` package, so I can properly format print my error
->   messages.
-> - I added the `Validate` method to my Config struct where I am checking if the
->   `interval` setting value is at least 1 minute (1m) and if the
->   `number_of_traces` setting value is greater or equal to 1. If that is not
->   true the Collector will generate an error during its startup process and
->   display the message accordingly.
+{{% alert title="Check your work" color="primary" %}}
+
+- Imported the `fmt` package to properly format print error messages.
+
+- Added the `Validate` method to the Config struct to check if the `interval`
```

Comment:
You've added blank lines between list entries, this spaces them out also in the generated pages, we don't need that extra spacing, so please remove the extra blank lines you added, here and elsewhere:

```suggestion
- Imported the `fmt` package to properly format print error messages.
- Added the `Validate` method to the Config struct to check if the `interval`
```